packages/redwood/src/index.ts

```diff
@@ -77,25 +82,37 @@ export async function build({
 
   debug('Running build command...');
   const { buildCommand } = config;
-
-  const found =
-    typeof buildCommand === 'string'
-      ? await execCommand(buildCommand, {
-          ...spawnOpts,
-          cwd: workPath,
-        })
-      : await runPackageJsonScript(
-          workPath,
-          ['vercel-build', 'build'],
-          spawnOpts
-        );
-
-  if (!found) {
-    throw new Error(
-      `Missing required "${
-        buildCommand || 'vercel-build'
-      }" script in "${entrypoint}"`
-    );
+  const frmwrkCmd = frameworks.find(f => f.slug === 'redwoodjs')?.settings
+    .buildCommand;
+  const pkgPath = join(workPath, 'package.json');
+  const pkg = JSON.parse(readFileSync(pkgPath, 'utf8')) as PackageJson;
+  if (hasScript('vercel-build', pkg)) {
+    debug(`Executing "yarn vercel-build"`);
+    await runPackageJsonScript(workPath, 'vercel-build', spawnOpts);
+  } else if (hasScript('build', pkg)) {
+    debug(`Executing "yarn build"`);
+    await runPackageJsonScript(workPath, 'build', spawnOpts);
+  } else if (buildCommand) {
```

Comment:
I think this has to be the first in this list, so it's considered first before `vercel-build` and `build`.

Reply:
You're right, I fixed it!